### PR TITLE
Tag IterativeSolvers.jl v0.3.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaMath/IterativeSolvers.jl](https://github.com/JuliaMath/IterativeSolvers.jl)
Release: [v0.3.0](https://github.com/JuliaMath/IterativeSolvers.jl/releases/tag/v0.3.0)
Travis: [Travis Build Status](https://travis-ci.org/JuliaMath/IterativeSolvers.jl/builds/204028679)
Diff: [vs v0.2.2](https://github.com/JuliaMath/IterativeSolvers.jl/compare/4558f41f71fa03fa0a331f58b6aa0e4658fc232d...d6092e65531aa25768175355e367577c513390e9)
`requires` vs v0.2.2: 
```diff
--- 0.2.2/requires
+++ 0.3.0/requires
@@ -1,3 +1,5 @@
-Compat
-julia 0.4
+julia 0.5-
 
+UnicodePlots
+RecipesBase
+SugarBLAS
```
cc: @andreasnoack

### CODE REVIEW

File: IterativeSolvers/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.5-
```
Comment (tkelman) [inline IterativeSolvers/versions/0.3.0/requires]: do you need to support prereleases?

**Comment by andreasnoack** [comment]
Not really. It has just been forever since last release.

**Comment by tkelman** [comment]
These methods are pretty questionable, I don't think they should be defined:
```
\(f::Function, b) = f(b)
*(f::Function, b) = f(b)
```

**Comment by andreasnoack** [comment]
I don't disagree but it has was decided by the maintainers long time ago and I'm not much involved in the development right now. There was a request for a tag since there has ben quite a few bug fixes since the latest release.

**Comment by andreasnoack** [comment]
I think there might be other problems in the package so let's hold off for now.

**Pull request closed without merging**